### DISCUSSION TITLE
Is there a way to change the domain of an existing docsearch project?

### DISCUSSION THREAD

**Discussion by Mister-Hope** [algolia/docsearch#2175] [Q&A] [answered]
I had applied mrhope.site previously as LDBQGQC8Q9, and now the domain changed to mister-hope.com

The website only gives me an apply page, which I guess I could get a new one though it, but what about the old one?  I am not finding a way to remove it.

Also I have a little discussion before, I am from VuePress team, for my team we serve different docs at different subdomain, so it would be great if we can add different subdomain in a single application. Some examples here:

- https://vuejs.press
- https://ecosystem.vuejs.press
- https://theme-hope.vuejs.press
- https://plugin-md-enhance.vuejs.press
- https://plugin-components.vuejs.press
- https://marketplace.vuejs.press

Applying theme one by one using the docsearch apply page could be annoying, and it makes us hard to maintain across different subdomains

So it would be great if we can have a way to apply to add/modify domains on existing applications created by docsearch, i.e. here:

![image](https://github.com/algolia/docsearch/assets/33315834/18a03d6d-99c3-4791-a70e-03e2470f577a)

Thanks if anyone in docsearch team could give a apply.

**Comment by Mister-Hope** [comment]
Update, I finally found the page and I think I can leave any unused apllication. 

![image](https://github.com/algolia/docsearch/assets/33315834/3612566c-7b3e-4955-af57-e55c68e072b5)

Just to make sure:

1. Once I leave mrhope application (old domain is no longer used) docsearch will have the application deleted automatically?
2. Is the first docsearch application is a historical one, and am I safe to leave?

Also I still want to have our subdomains managed under a single application, instead of a series. Applicate if anyone could give help.

**Comment by randombeeper** [comment, marked as answer]
Hi @Mister-Hope my apologies for the delay. Please reach me (jamesatalgolia) on our Discord at [Discord](https://alg.li/discord) if this is still a topic to discuss.

**Comment by randombeeper** [comment]
I'm going to close this discussion as we are trying to keep interactions to our Discord at https://alg.li/discord and we'll be happy to help you there! This discussion area is not well monitored.
